### ISSUE TITLE
Unlock watch mechanism

### ISSUE THREAD

**Issue by ajack2001my** [InfiniTimeOrg/InfiniTime#697]
I need everyone feedback on this. I constantly find that when I look at the watch, it is in one menu or another that was randomly registered while I was wearing the watch.

Can we have it such (suggestion only) where when we take or raise the wrist, it shows the watchface and any other touch input (while watchface is displayed before going to sleep) requires a "slide to unlock" before we can access the menus.

I am also open to any other suggestion. That said, it's only a discussion to gather ideas and hopefully, for a dev to take up as I don't code in "modern" languages... :)

**Comment by aemkai** [comment]
I would support this request. I have also the problem, that sometimes there is another screen, often also the alarm is disabled or brightness has changed by unwanted inputs.
Instead of slide to unlock I could imagine a double tap maybe better suited, because a unwanted sliding gesture  seems to occur often.

The issue appeared with the changed touch behaviour in 1.4.0, with Version 1.3.0 and prior there were rarely unwanted inputs.

**Comment by Riksu9000** [comment]
Duplicate of #395 and #618

**Comment by the-voidl** [comment]
This in combination with #395 would be a great improvement for the every day use of the watch!
I often find the clock in some unwanted state (like already mentioned by others). Since I also wear my clock in the shower and while doing sports, water and capacitive touchscreens are not the best friends - so I would also support some lock/unlock mechanism.

**Comment by Riksu9000** [comment]
Let's track this issue in #618
